src/container/AboutUsSection/AboutUsSection.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from "react"
```

Comment:
Chyba nie jest ci potrzebny ten import. Zwracaj uwagę na to co ci wypluwa konsola. Są tam informacje o nieużytych importach. Oczywiście usuwaj tylko swoje ;)